cluster/addons/metrics-server/auth-delegator.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: metrics-server:system:auth-delegator
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:auth-delegator
```

Comment:
this looks find but cc @deads2k

Reply:
this was reviewed in https://github.com/kubernetes-incubator/metrics-server/pull/4

Reply:
> this looks find but cc @deads2k

That's the role name. It allows the caller to hit tokenaccessreviews and subjectaccessreviews.